plugins/mozmenu.py

```diff
@@ -19,11 +19,29 @@
     "href": "https://mana.mozilla.org/wiki/display/DOPS/WTMO+Developer+Guide"
 }
 
+airflow_triage_guide = {
+    "name": "Airflow Triage Guide",
+    "category": "Mozilla",
+    "href": "https://mana.mozilla.org/wiki/display/DATA/Airflow+Triage+Process",
+}
```

Comment:
I think of this Mana link as canonical for triage, so adding this one here looks good. The two others below I think are too prone to iteration to include here. I'd rather we just link to the Mana page, and the Mana page is the one place where we'd be maintaining these links for bugzilla interaction.

Reply:
that's fair, I'll remove the other two then.